src/mlpack/methods/hmm/hmm.hpp

```diff
@@ -369,6 +436,30 @@ class HMM
 
 
  protected:
+  /**
+   * Given emission probabilities, computes forward probabilities at time t=0.
+   *
+   * @param emissionLogProb Emission probability at time t=0.
+   * @param logScales Vector in which the log of scaling factors will be saved.
+   * @return Forward probabilities
+   */
+  arma::vec ForwardAtT0(
+    const arma::vec& emissionLogProb,
+    double& logScales) const;
```

Comment:
```suggestion
      const arma::vec& emissionLogProb,
      double& logScales) const;
```
Just a style fix. :smile: